Rakefile

```diff
@@ -542,7 +543,7 @@ task :get_last_commit_date_shields do
   site_config = YAML.load_file("./config.yaml")
   for reldev in ['release', 'devel']
      for repo in ['bioc', 'data-experiment', 'workflows']
-        puts "writing badges for #{reldev} #{repo}"
+	puts "writing badges for #{reldev} #{repo}"
```

Comment:
would rather spaces for white space

Reply:
this happened when I did whitespace-cleanup